server/src/rooms/oldHub.ts

```diff
@@ -1,133 +1,124 @@
 // To do: make whole section no-video
 
-export const ASCII = {
-  id: 'ASCII',
-  displayName: 'The Hash Mark Dungeon',
-  shortName: '# dungeon',
-  description: 'The entire square tile you are standing on vibrates, and the world around you changes - you have entered an older time. The walls are made out of #, you can easily see the . at your feet, and your old powers of audio and video do not work here.</br></br>A row of twelve headstones stand in a line. They are all blank, except for the seventh, which is engraved with \'7 - A\'. Behind them stand three statues - a hobbit, a tree, and a crocodile. Each has a riddle engraved in its base.</br></br>A door behind you leads through the twisty passages, back to the [[Space Hanger->sfHub]].',
-  hasNoteWall: true,
-  hidden: true,
-  noMediaChat: true,
-  noteWallData: {
-    roomWallDescription: 'Do you want your possessions identified?',
-    noteWallButton: 'y/n/q',
-    addNoteLinkText: 'Tell a story',
-    addNotePrompt: 'Tell us of yet another sad/silly/surprising death you\'ve experienced in any roguelike.',
-    noteWallDescription: 'Tales of sad, silly, and surprising deaths in roguelikes of all kinds!'
+export const oldHubRoomData = {
```

Comment:
The thought of duplicating all these room IDs in the index file made me sad so I changed it to actually let me export them all at once